translations/ui-data-import/en_GB.json

```diff
@@ -219,7 +219,7 @@
     "editJobProfile": "Edit job profile",
     "run": "Run",
     "modal.jobProfile.run.header": "Are you sure you want to run this job?",
-    "modal.jobProfile.run.message": "You have selected the <b>\"{name}\"</b> job profile to <b>Run</b> the uploaded files",
```

Comment:
I think you'll have to get these fixed in Lokalise, iirc it overwrites

Reply:
These are just to cover me a little bit before Lokalize from master is pulled over. I think Lokalize updates based on what's in en.json.